lib/payload.json

```diff
@@ -20,6 +20,28 @@
       "description": "Speed (m/s)",
       "minimum": 0
     },
+    "depth": {
+      "type": "number",
+      "description": "Depth from the surface (cm)",
+      "minimum": 0
+    },
```

Comment:
I find this acceptable now. We ditched the relative depth thing and we ditched making the version identifiers available to the normalized payload. If anything, we're gonna work with state in the future.

I do agree with @cBashTN that we should use common units as much as possible. I think there's reason to differentiate between % and ppm for concentration, but I would prefer using meters for depth (and elevation).

Reply:
I went with `cm` because it made more sense to me just for soil probes, kind of the same way that `hPa` are used for atmospheric pressure instead of `Pa`. But in this case it is a global definition that may be used in any other context, so meters are more appropriate then.

Should we use one definition for `depth` and another one for `elevation` and set the ranges accordingly, or just use one definition (e.g, `vertical`) to cover all cases allowing negative and positive values?

Reply:
Okay, indeed we already use domain specific units (% and ppm). Let's keep things as domain specific as possible, so that we have soil depth in cm (and not generalize into `vertical`).